### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by C-Claus** [didix21/mdutils#31]
I have installed the mdutils on Python 3.5 with pip install.
On importing the module I get the following error:

```
Traceback (most recent call last):
  File "C:\Users\CClaus\projecten\Studytube\navisworks_manage.py", line 1, in <module>
    from mdutils import *
  File "C:\Python35\lib\site-packages\mdutils\__init__.py", line 2, in <module>
    from mdutils.mdutils import MdUtils
  File "C:\Python35\lib\site-packages\mdutils\mdutils.py", line 21, in <module>
    import mdutils.tools.Table
  File "C:\Python35\lib\site-packages\mdutils\tools\__init__.py", line 4, in <module>
    from .MDList import MDList
  File "C:\Python35\lib\site-packages\mdutils\tools\MDList.py", line 28
    md_list += self._add_new_item(item, f "{n_marker}")
                                                     ^
SyntaxError: invalid syntax

```

**Comment by didix21** [comment]
Which version have you installed ? Minimum python version is 3.6.

**Comment by C-Claus** [comment]
I am using:

`3.5.0 (v3.5.0:374f501f4567, Sep 13 2015, 02:27:37) [MSC v.1900 64 bit (AMD64)]`

Guess I will need to upgrade to 3.6. Or is there an easy workaround?

**Comment by didix21** [comment]
As you said, you have to upgrade to 3.6. Mdutils use f-strings and they are available as of Python 3.6. Thus, I think we can close this issue.
